mongo/crud_examples_test.go

```diff
@@ -451,3 +452,105 @@ func ExampleCollection_Watch() {
 		fmt.Println(changeStream.Current)
 	}
 }
+
+// Session examples
+
+func ExampleWithSession() {
+	var client *mongo.Client // assume client is configured with write concern majority and read preference primary
+
+	// Specify the DefaultReadConcern option so any transactions started through the session will have read concern
+	// majority.
+	// The DefaultReadPreference and DefaultWriteConcern options aren't specified so they will be inheritied from client
+	// and be set to primary and majority, respectively.
+	opts := options.Session().SetDefaultReadConcern(readconcern.Majority())
+	sess, err := client.StartSession(opts)
```

Comment:
I think we should defer a call to EndSession here and below

Reply:
Good catch. Done.